src/config/dynamic-theme-fixes.config

```diff
@@ -4082,10 +4082,10 @@ CSS
 ._3Ye_R {
     color: gray !important
 }
-[data-asset-intro-image] {
+[data-asset-intro-image], [data-asset-intro-image-light] {
     background-image: url(/img/intro-connection_c98cc75f2aa905314d74375a975d2cf2.jpg) !important;
 }
-._11ozL {
+._1QMFu {
```

Comment:
Changing every month? Let's see if we can use a `>` strict CSS3 to target this CSS to make it more reliabler in the next updates for whatsapp. Related what you did with google 9 dots in the right above corner?

Reply:
Saw inside class nice canvas with aria-label="Scan me!" but making border for it doesn't work and CSS still doesn't work for parents. 😉 but who knows... maybe I'll find some solution in the future.

Reply:
Let's not merge it yet I will have a look at it today to get a more global solution to this.

Reply:
`html[dir] .landing-main > :first-child > :nth-child(2) > :first-child`